localstack/config.py

```diff
@@ -1041,6 +1041,9 @@ def populate_edge_configuration(
 # Show exceptions for CloudFormation deploy errors
 CFN_VERBOSE_ERRORS = is_env_true("CFN_VERBOSE_ERRORS")
 
+# Set the timeout for CloudFormation resources
+CFN_PER_RESOURCE_TIMEOUT = int(os.environ.get("CFN_PER_RESOURCE_TIMEOUT") or 150)
```

Comment:
We should probably increase the default as well since we know that nested stacks are already a problem. Perhaps `300` (5 minutes)?